testing/sdk_plan.py

```diff
@@ -25,6 +27,16 @@ def get_recovery_plan(service_name):
     return get_plan(service_name, "recovery")
 
 
+def list_plans(service_name):
+    def fn():
+        output = sdk_api.get(service_name, '/v1/plans')
+        try:
+            return output.json()
+        except:
+            return False
+    return shakedown.wait_for(fn)
```

Comment:
Maybe use the `retry` library as we're gradually reducing dependency on shakedown?